site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -875,6 +901,128 @@ This strategy instructions FusionAuth when to append a unique suffix to the user
 * `Always` - Always append a unique suffix even when the requested username is not in use.
 * `OnCollision` - Only append a unique suffix when the requested username is in use.
 
+[field]#tenant.webAuthnConfiguration.bootstrapWorkflow.authenticatorAttachmentPreference# [type]#[String]# [optional]#Optional# [default]#defaults to `crossPlatform`# [since]#Available since 1.41.0#::
+Determines the authenticator attachment requirement for WebAuthn passkey registration when using the bootstrap workflow. The possible values are:
++
+* `any` - An authenticator with any attachment modality will be allowed during registration.
+* `crossPlatform` - Only authenticators with the `cross-platform` attachment modality will be allowed during registration. These are also referred to as "roaming" authenticators.
+* `platform` - Only authenticators with the `platform` attachment modality will be allowed during registration.
+
++
```

Comment:
I could not get this to format properly. There are other examples I found where it is a bulleted list followed by one of the `enterprise_feature` blurbs, but this case has a list followed by a recommendation followed by a `premium_feature` blurb followed by an `enterprise_feature` blurb. The enterprise blurb is correct, but I could not get the recommendation and premium feature blurb in the right spot.

![image](https://user-images.githubusercontent.com/3409780/201366956-e3765df5-71f4-4796-85ec-06ac65047dcf.png)